docs/configuration.rst

```diff
@@ -173,6 +173,38 @@ search_terms
     Then ``search_terms`` will be available in the browse template context to help
     the user understand how their search will work.
 
+sorting
+    The default browse template will include sort controls on the column headers
+    for columns that are sortable.
+
+    If the second item in the column entry for a column is not a valid specification
```

Comment:
This reads better to me if we change the first 'column' to `columns`, so that I know it refers to the columns attribute referenced earlier.